src/vue/backend/src/views/children/importedAgents.vue

```diff
@@ -0,0 +1,61 @@
+<template>
```

Comment:
`importedAgents` and `unimportedAgents` are nearly identical, save some event handlers. I think this could be done a little more DRY. No need to change now, but let's think about it.